packages/oid4vci-demo-frontend/src/configs/sphereon.json

```diff
@@ -131,6 +131,63 @@
         }
       }
     },
+    "SSICredentialsLandingPage": {
```

Comment:
Shouldn't this be in a file called knb.json? It has little to do with the generic Sphereon demo.

I am okay with leaving it for now if it doesn't interfere with the generic Sphereon demo, but what is the path/url for this page then?

Reply:
it's now in a file called knb.json as indeed that makes more sense